diff --git a/bootstrap.cmd b/bootstrap.cmd
new file mode 100755
index 0000000..d4b8774
--- /dev/null
+++ b/bootstrap.cmd
@@ -0,0 +1,7 @@
+extproc sh
+
+(cd t && attrib -r "*")
+
+export COMSPEC=/bin/sh
+
+./bootstrap.sh "$@"
diff --git a/configure.cmd b/configure.cmd
new file mode 100755
index 0000000..50c68fc
--- /dev/null
+++ b/configure.cmd
@@ -0,0 +1,3 @@
+extproc sh
+
+./configure --prefix=/usr "$@"
