diff --git a/Makefile.am b/Makefile.am
index 143308a..8501e2a 100644
--- a/Makefile.am
+++ b/Makefile.am
@@ -72,7 +72,7 @@ generated_file_finalize = $(AM_V_at) \
 # For some tests or targets, we need to have the just-build automake and
 # aclocal scripts avaiable on PATH.
 extend_PATH = \
-  { PATH='$(abs_builddir)/t/wrap'$(PATH_SEPARATOR)$$PATH && export PATH; }
+  { PATH='$(abs_builddir)/t/wrap$(PATH_SEPARATOR)'$$PATH && export PATH; }
 
 # The master location for INSTALL is lib/INSTALL.
 # This is where "make fetch" will install new versions.
diff --git a/t/Makefile.inc b/t/Makefile.inc
index b67f3a5..e4fe06c 100644
--- a/t/Makefile.inc
+++ b/t/Makefile.inc
@@ -64,9 +64,9 @@ AM_TESTS_ENVIRONMENT += stderr_fileno_=9; export stderr_fileno_;
 # in case it is given with a relative name containing no slashes.
 AM_TESTS_ENVIRONMENT += \
   if test $(srcdir) != .; then \
-    PATH='$(abs_srcdir)/%D%/ax'$(PATH_SEPARATOR)$$PATH; \
+    PATH='$(abs_srcdir)/%D%/ax$(PATH_SEPARATOR)'$$PATH; \
   fi; \
-  PATH='$(abs_builddir)/%D%/ax'$(PATH_SEPARATOR)$$PATH; \
+  PATH='$(abs_builddir)/%D%/ax$(PATH_SEPARATOR)'$$PATH; \
   export PATH;
 
 # Hand-written tests.
